models/spring-ai-openai/src/test/java/org/springframework/ai/openai/chat/client/OpenAiChatClientIT.java

```diff
@@ -195,6 +195,21 @@ void beanOutputConverter() {
 		assertThat(actorsFilms.actor()).isNotBlank();
 	}
 
+	@Test
+	void beanOutputConverterNativeStructuredOutput() {
+
+		// @formatter:off
+		ActorsFilms actorsFilms = ChatClient.create(this.chatModel).prompt()
+				.advisors(AdvisorParams.ENABLE_NATIVE_STRUCTURED_OUTPUT)
```

Comment:
Do you think having a top level chat client property or even an option at ChatClient level to enable the native structured output be better than Advisor param?